scipy/stats/tests/test_morestats.py

```diff
@@ -23,6 +23,7 @@
 try:
     import matplotlib.pyplot as plt
     have_matplotlib = True
+    matplotlib.rcParams['backend'] = 'Agg'
```

Comment:
The rcParam has to be changed before importing pyplot.

Reply:
Split the import into two statements with the parameter setting sandwiched in the middle.
```
    import matplotlib
    matplotlib.rcParams['backend'] = 'Agg'
    import matplotlib.pyplot as plt
    have_matplotlib = True
```